docs/authoring.md

```diff
@@ -109,6 +114,8 @@ Consuming a C#/WinRT component from a C++/WinRT desktop application is supported
 
 - For project references, you also currently need a reference to the component's generated WinMD. The WinMD can be found in the output directory of the authored component's project. To add both the project and WinMD references, right-click on the native project node, and click **Add** -> **Reference**. Select the C#/WinRT component project under the **Projects** node and the generated WinMD file from the **Browse** node.

```

Comment:
Maybe instead of the note below, preface this bullet with "If your authored component is built with .NET 5, you also need a reference to the WinMD from referencing C++ apps?